charts/ceph-csi-rbd/values.yaml

```diff
@@ -143,6 +143,8 @@ provisioner:
   deployController: true
   # Timeout for waiting for creation or deletion of a volume
   timeout: 60s
+  # cluster name to set on the RBD image
+  clustername: "cluster1"
```

Comment:
this is the name of the Kubernetes cluster, right? Ideally explain that, and maybe use `k8s-cluster-1` or something. Many cloud providers use `-` as separation between clusters/regions/...